manageperson.rb

```diff
@@ -0,0 +1,35 @@
+def create_teacher
+  print "Teacher's Specialization: "
+  specialization = gets.chomp
+  print "Teacher's age: "
+  age = gets.chomp.to_i
+  print "Teacher's name: "
+  name = gets.chomp
+
+  teacher = Teacher.new(age, name, specialization, parent_permission: true)
```

Comment:
- [ ] @emhamza and @Jeddaa  Great work creating a module to handle the  teacher addition, however, I would suggest you move this line to your `create_person` method, this will ensure the app follow `Single responsibility rule` where each function is dedicated to a specific task alone

Reply:
Please explain what you are trying to say. You current feedback is  simply confusing us! :)